jwt-wallet.go

```diff
@@ -113,3 +126,41 @@ func handleToken(kong *pdk.PDK, tokenString string) (*jwt.Token, error) {
 	}
 	return token, nil
 }
+
+func verifyAddress(addr string, pubKey string, hrp string) bool {
+	keyB64 := strings.Split(pubKey, ",")[0]
+	keyBytes, err := base64.RawURLEncoding.DecodeString(keyB64)
+
+	if err != nil {
+		fmt.Printf("Could not decode public key")
```

Comment:
Unless there is a reason to avoid it, I'd log the `err` as well.

Reply:
addressed and now logging